src/HttpCache/VarnishPurger.php

```diff
@@ -48,10 +48,10 @@ public function purge(array $iris)
             return sprintf('(^|\,)%s($|\,)', preg_quote($iri));
         }, $iris);
 
-        $regex = isset($parts[1]) ? sprintf('(%s)', implode(')|(', $parts)) : $parts[0];
+        $regex = count($parts) > 1 ? sprintf('(%s)', implode(')|(', $parts)) : array_shift($parts);
```

Comment:
Perf are better with `current()` (or `array_pop()` for example). But I prefer `reset()`, it's more safe.